dev/com.ibm.ws.kernel.boot.ws-server/publish/bin/server

```diff
@@ -1055,6 +1055,10 @@ javaCmd()
       # bad checkpoint arg; sleep to allow the message to go to tail
       sleep 2
     fi
+    if [ $rc -eq 74 ] && [ -f "${X_LOG_DIR}/checkpoint/checkpoint.log" ]; then
```

Comment:
I assume `[ $rc -eq 74 ]` is guaranteed by jdk to be returned when criu dump failed?
If `[ $rc -eq 74 ]` but checkpoint.log not created, should some other message be cat-ed out?

Reply:
74 is what Liberty returns as the exit code if a `org.eclipse.openj9.criu.SystemCheckpointException` is thrown.  See `io.openliberty.checkpoint.internal.criu.CheckpointFailedException.Type.SYSTEM_CHECKPOINT_FAILED` enum and calls to `io.openliberty.checkpoint.internal.criu.CheckpointFailedException.getErrorCode()`. The `SystemCheckpointException` is thrown by the JVM for failures in CRIU to checkpoint.  Generally this will result in a `checkpoint.log` with content.  But there are cases where CRIU fails much earlier than the point where it even creates the `checkpoint.log`

I'm not sure what other message we should display when there is no `checkpoint.log` to look at.  There will be other messages earlier in the logs that indicate a checkpoint has failed.  For now I think we can just depend on these earlier messages for information (as little as it may be).